packages/devtools_app/lib/src/utils.dart

```diff
@@ -966,7 +966,8 @@ extension LogicalKeySetExtension on LogicalKeySet {
     return sortedKeys.map((key) {
       if (_modifiers.contains(key)) {
         if (isMacOS && key == LogicalKeyboardKey.meta) {
-          return '⌘';
+          // TODO(elliette) Switch back to using ⌘ once supported on web.
```

Comment:
When there is a specific bug, you can either write `TODO(elliette): some description. See <github link to issue>` Or you can write the todo specifically attached to the issue link `TODO(<github link to issue): use ⌘ instead of command`

Reply:
Great, thanks!